pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -378,16 +378,26 @@ func (os *OpenStack) GetMaxVolLimit() int64 {
 
 // diskIsAttached queries if a volume is attached to a compute instance
 func (os *OpenStack) diskIsAttached(instanceID, volumeID string) (bool, error) {
-	volume, err := os.GetVolume(volumeID)
-	if err != nil {
-		return false, err
-	}
-	for _, att := range volume.Attachments {
-		if att.ServerID == instanceID {
-			return true, nil
+	var foundAttachment bool = true
```

Comment:
Is this the correct default? If I understand the code correct, this way ` foundAttachment` would be always `true`, or did I miss something?

Reply:
Yeah apologies I think I confused this with the behaviour of EachPage below for some reason. I'll update and try again now.